src/test/resources/JDI_ex8_metalsColorsDataSet.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
You should not modify this file, use it as it is.

Reply:
Left file as it should be and transform it to POJO a bit different way.